install_config/aggregate_logging.adoc

```diff
@@ -855,7 +855,18 @@ spec:
 ----
 <1> This label should uniquely identify a replica with a single node that bears that
 label, in this case `logging-es-node=1`. Use the `oc label` command to apply
-labels to nodes as needed.
+labels to as many nodes as needed.
+
+For example, if your deployment has three infrastructures, you could add labels for those
+infrastructures as follows:
+----
+oc label node <nodename1> logging-es-node=1
```

Comment:
s/oc/$ oc
(three places)

Reply:
Thanks. Added $ before "oc" in three places.